.gitlab/ci/staging.yml

```diff
@@ -0,0 +1,129 @@
+graph-create-instances:
```

Comment:
@esharf can you please have a look at the CI changes as well? (especially under `.gitlab/ci`)